src/test/java/uk/gov/companieshouse/pscfiling/api/service/impl/PscWithIdentificationFilingServiceImplTest.java

```diff
@@ -54,14 +58,8 @@ void setUp() {
     }
 
     @Test
-    void createFiling() {
-        testService.createFiling(filing);
-        verify(filingRepository).save(filing);
-    }
-
-    @Test
-    void getFiling() {
-        testService.getFiling(FILING_ID);
+    void get() {
```

Comment:
Do we save() as in IndividualFilingServiceImplTest?

Reply:
Yes you're right, I'll rename `getFiling()` to `get()` to match the method under test, and remove `createFiling()` since it's duplicating the test in `save()` lower down.